server/models/Group.js

```diff
@@ -483,4 +486,13 @@ export default function(Sequelize, DataTypes) {
   });
 
   return Group;
+
+  function isThankDonationEnabled(GroupId) {
+    return models.Notification.findOne({where: {
+      channel: 'twitter',
+      type: activities.GROUP_TRANSACTION_CREATED,
```

Comment:
this isn't a `group_transaction_created`. Can we create a new type of activity for it?